lib/inherit.js

```diff
@@ -84,7 +79,14 @@ function override(base, res, add) {
 
                         return res;
                     };
-                result.__base = baseMethod;
+
+                result.__base = function() {
+                    var baseMethod = base[name]
```

Comment:
* Yes, it's ambiguous as well as javascript has no support of classes up to ES2015.
  * Yes, it does. As in languages that support supercalls.
  * In case of next tick you have to save a link to `this.__base` manually. I agree it's not an ideal solution but do you really do async supercalls often?
  * It should be able being mockable after #18

`this.__parent.<method>(arg1, arg2, ...);` looks like too verbose compared to `this.__base(arg1, arg2, ...)`. It makes you repeat a method name from a base class. In languages that support supercalls natively you don't have to repeat it.